sunpy/timeseries/sources/goes.py

```diff
@@ -105,7 +108,6 @@ def peek(self, title="GOES Xray Flux"):
 
         axes.fmt_xdata = matplotlib.dates.DateFormatter('%H:%M')
         figure.autofmt_xdate()
-        figure.show()
 
```

Comment:
Don't want to return the figure like in the other functions?

Reply:
yes.